geode-docs/tools_modules/geode_for_redis.html.md.erb

```diff
@@ -186,7 +186,7 @@ Could not connect to Redis at 127.0.0.1:6379: Connection refused
 | HINCRBY | HINCRBYFLOAT | HKEYS | HLEN |
 | HMGET | HMSET | HSCAN **[3]** | HSET |
 | HSETNX | HSTRLEN | HVALS | INCR |
-| INCRBY | INCRBYFLOAT | INFO **[4]** | KEYS |
+| INCRBY | INCRBYFLOAT | INFO **[4]** | KEYS | LINDEX |
```

Comment:
I think that the formatting of this section will get messed up if you just add another element like this, since it's currently set up as a table with 4 columns. Adding LINDEX here will (I think) add a fifth column with only LINDEX in it. This makes updating the list of commands a nightmare, since adding one means you need to adjust every command that comes after it in the list. We probably need a better way of handling this section if we're planning to modify it frequently. Maybe breaking it up by command type? That's work for another ticket though, I think.

Luckily (for this PR, at least) if you add LPUSH, LPOP and LLEN to the list of supported commands too, then not only will the docs be up to date, but you can just add one extra row of 4 elements below this one and not have to worry about affecting the rest of the table.